<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>824</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QLabel" name="label_bangsach">
         <property name="text">
          <string>Sách</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_bangdg">
         <property name="text">
          <string>Độc giả</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTableWidget" name="table_sach">
         <column>
          <property name="text">
           <string>Mã sách</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tên Sách</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Số lượng</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTableWidget" name="table_docgia">
         <column>
          <property name="text">
           <string>Mã Độc Giả</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tên Độc Giả</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_masach">
           <property name="text">
            <string>Mã Sách    </string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_timsach">
           <property name="text">
            <string>Tìm Kiếm</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_masach"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_trangthai">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Trạng Thái</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_tinhtrang"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_madocgia"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_docgia">
           <property name="text">
            <string>Mã Độc Giả</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_timdg">
           <property name="text">
            <string>Tìm Kiếm</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_them">
       <property name="text">
        <string>Thêm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_update">
       <property name="text">
        <string>Cập Nhật</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_xoa">
       <property name="text">
        <string>Xoá</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_tim_muon_tra">
       <property name="text">
        <string>Tìm kiếm Mượn/Trả</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tbl_muontra">
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Mã Độc Giả</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tên Độc Giả</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Mã Sách</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tên Sách</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ngày Mượn</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ngày Trả</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Trạng Thái</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
